data/schedules_report/monthly.yaml

```diff
@@ -0,0 +1,18 @@
+name: testing schedule 1
+description: description of testing schedule 1
+filter:
+    - Events
+    - Policy
+    - Policy Events for Last Week
+active: True
+timer:
+    run: Monthly
+    hours: 2 Months
```

Comment:
Question : Same here .. can it be run in 1 Month and weekly report in 1 week instead of 2 weeks ?